docs/source/contrib-setup.rst

```diff
@@ -13,18 +13,15 @@ Decide whether you'll work on a clone or a fork.
 
 Choose a clone if:
 
-- You are a bioconda team member (post on `issue #1`_ if you'd like to
-  join). Team members have write access to branches other than the master
-  branch.
-
-- You want to have other team members make changes directly to your branch
+- You are a bioconda team member. Team members have write access to branches
```

Comment:
Hmm, at the moment we're not requiring anything on `bulk`. Shouldn't we just directly restrict who can merge to it? We can then document that here.

Reply:
Update, people can still merge once tests pass on `bulk`. They weren't required previously for some reason.

Reply:
From reading through the documentation it seems that it's not possible to completely prevent members from merging PRs to bulk if the CI tests pass. That explains how I could sometimes merge things into `bulk` 6 months ago. I expect that's a risk we'll have to take for the time being. BTW, we should add CODEOWNERS for the CI configuration stuff, just to prevent anyone from accidentally mucking with it. I'll try to make a PR for that today and see if I can use it to prevent any modifications outside of `recipes/`, to prevent things like the current `/multiphate`.

Reply:
I not afraid people merging passed PRs to bulk, it's as risky as merging passed PRs to master. What is a problem is merging failed into bulk, which should only be allowed for admins.

> BTW, we should add CODEOWNERS for the CI configuration stuff, just to prevent anyone from accidentally mucking with it. I'll try to make a PR for that today and see if I can use it to prevent any modifications outside of recipes/, to prevent things like the current /multiphate.

I thought that can be only be used to ping people if certain files are modified. Does it prevent people from modifying files? Can the bot fail linting iff user != bioconda/core and modified-files != recipes/* | blacklist ?

Reply:
Codeowners have reviews requested automatically, so yes at least we get notified.

Reply:
I'm ditching the CODEOWNERS idea, since there's no way I can find to whitelist the blacklist and I don't want to get spammed every time someone needs to update that.